### PR TITLE
Adding virtual machine specs to support Kubevirt

### PR DESCRIPTION
**Pull request by vpinisetti-px**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes # https://portworx.atlassian.net/browse/PA-1698

**Special notes for your reviewer**:
```
[root@vm-vpinisetti:~/kubevirt/working]# kc get vm -A
NAMESPACE       NAME                          AGE     STATUS    READY
default         centos7-um6b40cbo5mwg5w5      3d20h   Running   True
test2           ubuntu-mysql-vm               32h     Running   True
test3           vm-ubuntu-containerdisk       149m    Running   True
test5           vm-ubuntu-containerdisk-pvc   47m     Running   True
ubuntu-bionic   ubuntu-bionic                 3d23h   Running   True
ubuntu-pvc      ubuntu-pvc-vm                 3d23h   Running   True
[root@vm-vpinisetti:~/kubevirt/working]#
```

### CODE REVIEW

File: drivers/scheduler/k8s/specs/kubevirt/vm-ubuntu-containerdisk.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        - name: cloudinitvolume
+          cloudInitNoCloud:
+            userData: |-
+              #cloud-config
+              chpasswd:
+                list: |
+                  ubuntu:ubuntu
+                  root:toor
+                expire: False
+
```
Comment (kshithijiyer-px) [inline drivers/scheduler/k8s/specs/kubevirt/vm-ubuntu-containerdisk.yaml:48]: Blank line at the end of the file

**Comment by mkoppal-px** [comment]
@vpinisetti-px Do you want to name the app as `kubevirt` ?

**Comment by vpinisetti-px** [comment]
> @vpinisetti-px Do you want to name the app as `kubevirt` ?

I feel that is right name as we really dont worry about the application inside the vm. Please suggest incase you have anything in your mind.

**Comment by mkoppal-px** [comment]
`kubevirt-multi-vm` can be a good name

File: drivers/scheduler/k8s/specs/kubevirt/vm-ubuntu-containerdisk.yaml (hunk 1 above)
Comment (vpinisetti-px) [reply drivers/scheduler/k8s/specs/kubevirt/vm-ubuntu-containerdisk.yaml]: Done

**Comment by vpinisetti-px** [comment]
> `kubevirt-multi-vm` can be a good name

Done

**Comment by vpinisetti-px** [comment]
@mkoppal-px / @kshithijiyer-px Is it ok to publish to docker registry name in the spec `docker.pwx.dev.purestorage.com/portworx/ubuntu-container-disk:20.0` ?

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**